adhocracy-plus/assets/js/app.js

```diff
@@ -59,16 +60,14 @@ function init () {
   // This function adds required classes to iframes added by ckeditor
   $('.rich-text iframe').addClass('ck_embed_iframe')
   $('.ck_embed_iframe').parent('div').addClass('ck_embed_iframe__container')
+
+  // Initialize observers for dashboard dropdowns
+  initDashboardAccordeon()
 }
```

Comment:
I think it might be better to only load this on dashboard as it's not needed anywhere else, what do you think?